globus_automate_client/cli/helpers.py

```diff
@@ -37,8 +38,15 @@
 
 def get_http_details(result: Union[GlobusHTTPResponse, GlobusAPIError]) -> str:
     if isinstance(result, GlobusHTTPResponse):
-        base_request = result.data["request"]
-        response_status_code = result.data["status_code"]
+        if isinstance(result._response, requests.Response):
+            base_request = result._response.request
+        elif (
+            result._wrapped and isinstance(result._wrapped._response, requests.Response)
+        ):
+            base_request = result._wrapped._response.request
+        else:
+            return "HTTP details are unavailable"
+        response_status_code = result.http_status
```

Comment:
Nice, I was thinking I might show up here and be able to suggest swapping to `http_status` or something else to that effect. I think this change looks good. Would it be helpful to think about a "debug output" mode on `GlobusHTTPResponse` for cases like this? I feel like `str(response)` being just the JSON could maybe be more helpful, and do something more like this function.

Reply:
Thanks! Is the "debug output" something to change in this repo or in the SDK? If it's in the SDK I'd like to pass that feedback to the SDK team.

Reply:
Yeah, that's what I was getting at. I'll float that in our next SDK catch-up.

Reply:
Okay, great. I'm marking this as 'resolved' but please re-open as needed.